Podfile

```diff
@@ -30,6 +30,7 @@ target "Amplify" do
     inherit! :complete
     use_frameworks!
 
+    pod "AWSCore", "~> #{AWS_SDK_VERSION}"
```

Comment:
is this because we will remove AWSMobileClient (below line 34) soon? so we need a direct dependency on AWSCore? Do we need to update the Podspec at some point?

Reply:
Yes that is right. I updated AWSPluginsCore.podspec.